From e687d88e5b7ae7a74829e4d00374518b364bd79c Mon Sep 17 00:00:00 2001
Message-Id: <e687d88e5b7ae7a74829e4d00374518b364bd79c.1414996573.git.chang-joon.lee@intel.com>
In-Reply-To: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
References: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Fri, 31 Oct 2014 11:41:32 +0000
Subject: [PATCH 07/10] FOR_UPSTREAM [VPG]: drm/i915: Remove unused variable
 in cherryview_irq_handler

The following commit added an unused variable:

commit c0e8763c23a76111b7fa6392705f8b0aa4ca837a
Author: Vandita Kulkarni <vandita.kulkarni@intel.com>
Date:   Tue Oct 14 18:01:43 2014 +0530

    FOR_UPSTREAM [VPG]: drm/i915: Add support for chv HDMI LPE audio

For: GMINL-1778
Change-Id: I369d540a8d8e4fa8255b54036cf9cfb2f559ea3e
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/i915_irq.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_irq.c b/drivers/gpu/drm/i915/i915_irq.c
index 580b28d..bebfc68 100644
--- a/drivers/gpu/drm/i915/i915_irq.c
+++ b/drivers/gpu/drm/i915/i915_irq.c
@@ -2229,7 +2229,6 @@ static irqreturn_t cherryview_irq_handler(int irq, void *arg)
 	struct drm_i915_private *dev_priv = dev->dev_private;
 	u32 master_ctl, iir;
 	irqreturn_t ret = IRQ_NONE;
-	int lpe_stream;
 
 	for (;;) {
 		master_ctl = I915_READ(GEN8_MASTER_IRQ) & ~GEN8_MASTER_IRQ_CONTROL;
-- 
1.7.9.5

